### DISCUSSION TITLE
Logbook entries after loading backup

### DISCUSSION THREAD

**Discussion by dijkr** [albar965/littlenavmap#1086] [Q&A] [answered]
Hello,

I've made a back-up of th %appdata% location for Navmap. After replacing this file, I can see the summary of the logbook, but the flights are not listed at logbook within the information panel. Does that make sense? Would it be possible to backup this data, including flight tracks too? I hope I can still recover this, if not, it's nice to know for the future how to do that right. 

This is a quote from the manual 27. Logbook
`Each logbook entry has a reference using the full path to the used flight plan and aircraft performance files. Keep in mind that these references break if the files are moved or renamed. Update them manually if needed.`

Thanks in advance!

**Comment by albar965** [comment, marked as answer]
LNM refers/links to the used flight plan and aircraft performance files. If you rename or remove these file the links are disabled. But this is no issue because both are additionally attached to the logbook entries.

You can backup all logbook entries including the attached (not linked) files by either copying the `little_navmap_logbook.sqlite` file or exporting to CSV. But keep the three options checked to export the attached files:
![grafik](https://github.com/albar965/littlenavmap/assets/15254956/c575d617-8dec-495c-8c1b-0c9b5d1ae7a9)

Maybe a search option is enabled if you see statistics but no logbook entries in the result table of the logbook.

Alex

**Reply by dijkr** [reply]
Thanks a lot for your effort and support! The Logbook search option is showing all the entries. Then, I can even use "Show aircraft trail". Its all there. I started to fly again, after two years, and this makes me very happy. Thanks again :-)

**Reply by albar965** [reply]
You're welcome. :slightly_smiling_face:
